doc/glossary.rst

```diff
@@ -0,0 +1,1645 @@
+        with number of columns fixed after :term:`fitting`.
+
+        If the estimator was not already :term:`fitted`, calling this method
+        should raise a :class:`exceptions.NotFittedError`.
+
+.. _glossary_parameters:
+
+Parameters
+==========
+
+These common parameter names, specifically used in estimator construction
+(see concept :term:`parameter`), sometimes also appear as parameters of
+functions or non-estimator constructors.
+
+.. glossary::
+
+    ``class_weight``
+        Used to specify sample weights when fitting classifiers as a function
+        of the :term:`target` class.  Where :term:`sample_weight` is also
+        supported and given, it is multiplied by the ``class_weight``
+        contribution. Similarly, where ``class_weight`` is used in a
+        :term:`multioutput` (including :term:`multilabel`) tasks, the weights
+        are multiplied across outputs (i.e. columns of ``y``).
+
+        By default all samples have equal weight such that classes are
+        effectively weighted by their their prevalence in the training data.
+        This could be achieved explicitly with ``class_weight={label1: 1,
+        label2: 1, ...}`` for all class labels.
+
+        More generally, ``class_weight`` is specified as a dict mapping class
+        labels to weights (``{class_label: weight}``), such that each sample
+        of the named class is given that weight.
+
+        ``class_weight='balanced'`` can be used to give all classes
+        equal weight by giving each sample a weight inversely related
+        to its class's prevalence in the training data:
+        ``n_samples / (n_classes * np.bincount(y))``.
+
+        For multioutput classification, a list of dicts is used to specify
+        weights for each output. For example, for four-class multilabel
+        classification weights should be ``[{0: 1, 1: 1}, {0: 1, 1: 5}, {0: 1,
+        1: 1}, {0: 1, 1: 1}]`` instead of ``[{1:1}, {2:5}, {3:1}, {4:1}]``.
+
+        The ``class_weight`` parameter is validated and interpreted with
+        :func:`utils.compute_class_weight`.
+
+    ``cv``
+        Determines a cross validation splitting strategy, as used in
+        cross-validation based routines. ``cv`` is also available in estimators
+        such as :class:`multioutput.ClassifierChain` or
+        :class:`calibration.CalibratedClassifierCV` which use the predictions
+        of one estimator as training data for another, to not overfit the
+        training supervision.
+
+        Possible inputs for ``cv`` are usually:
+
+        - An integer, specifying the number of folds in K-fold cross
+          validation. K-fold will be stratified over classes if the estimator
+          is a classifier (determined by :func:`base.is_classifier`) and the
+          :term:`targets` may represent a binary or multiclass (but not
+          multioutput) classification problem (determined by
+          :func:`utils.multicla
```

Comment:
`` -> ` after n_features (please ignore the same message above)

Reply:
I don't understand what you're asking for.

Reply:
In L1546, a redundant backtick after :term:`n_features` (easy to spot through the diff). Not sure why my comment goes into the wrong place. Sorry for such a mess.